apps/posts/src/views/PostAnalytics/components/PostAnalyticsHeader.tsx

```
@@ -10,63 +10,102 @@ interface PostWithPublishedAt extends Post {
-            }
-        </div>
+                    {!isPostLoading &&
+                        <div className='flex items-center gap-6'>
+                            {post.feature_image &&
+                                <div className='h-[82px] w-[132px] rounded-md bg-cover' style={{
+                                    backgroundImage: `url(${post.feature_image})`
+                                }}></div>
+                            }
+                            <div>
+                                <H1 className='-ml-px min-h-[35px] max-w-[920px] indent-0 leading-[1.2em]'>
+                                    {post && post.title}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)